test/dlc_tests/conftest.py

```diff
@@ -421,14 +425,34 @@ def pytest_configure(config):
 
 
 def pytest_runtest_setup(item):
+    """
+    Handle custom markers and options
+    """
+    # Handle quick check tests
+    quick_checks_opts = [mark for mark in item.iter_markers(name="quick_checks")]
+    # Skip quick checks on PR sanity tests, as they will run in separate build-independent job
+    if os.getenv("TEST_TYPE") == "sanity" and os.getenv("BUILD_CONTEXT") == "PR":
```

Comment:
nit: Recommend using `is_pr_context()` from `test_utils/__init__.py` instead of directly checking the "BUILD_CONTEXT" env variable, in order to modularize the check.

Reply:
sure thing, can make this change